index.html

```diff
@@ -195,35 +216,42 @@ <h2>CONTACT US</h2>
                 <br />
                 <input type="email" id="email" name="email" placeholder="jeremy@gmail.com" required /><br />
                 <br />
-                <textarea name="comments" id="comments" placeholder="Enter text here" maxlength="500" rows="8" cols="40" required></textarea>
-                <input class="button2" type="submit" value="See Project" />
+                <textarea name="comments" id="comments" placeholder="Enter text here" maxlength="500" rows="8" cols="46" required></textarea>
```

Comment:
- [ ] maybe you didn't notice. The mobile version is supposed to still look ok. kindly align the input fields
![Screenshot (214)_LI](https://user-images.githubusercontent.com/65298048/177862149-0fe22df9-8aad-4f9f-8245-f62211a9accc.jpg)